### DISCUSSION TITLE
Audit of Accounts - drafting intent

### DISCUSSION THREAD

**Discussion by Aaron-XReg** [MinistryFinancialServices/Virtual-Asset-Amendment-Consultation#32] [General]
S. 31 Audit of Accounts has been proposed to be amended to include the following language:

"Every licensee shall have its accounts audited annually or at such other times as the Authority may require by an auditor who shall be a chartered accountant, certified public accountant or other professionally qualified accountant approved by the Authority."

Is the intention that only "other professionally qualified" accountants require approval by the authority? I doubt this is the intention but it may assist smaller firms to be able to use off-Island accounting firms, at least for a period of time, because currently it can be difficult to get CIMA approved auditors to audit VASP firms.

**Comment by lkoseland** [comment]
It is proposed that the auditor be approved by the Authority. Given the limited number of auditors on Island specializing in the audit of digital assets, it is suggested that any auditor with the required expertise is permitted to audit the accounts of a virtual asset service provider (VASP) if they are regulated in a jurisdiction with an equivalent supervisory regulatory regime for auditors as the Islands. The Authority could apply the Regulatory Policy on the Approval of an Auditor for a Regulated Institution to VASPs and disapply the requirement for local audit sign-off at paragraph 2.8. 
Until there is a sufficient number of auditors on Island to prevent a bottle neck from occurring for a VASP to fulfill the regulatory requirement of having an annual audit, the above approach could provide comfort for the Authority to ensure that the accounts of VASPs are adequately audited by experts in the field.
